media/editor/dataInspectorProperties.tsx

```diff
@@ -1,3 +1,52 @@
+/** Reads a ULEB128 at offset 0 from the buffer. */
+const getULEB128 = (arrayBuffer: ArrayBuffer) => {
+	const buf = new Uint8Array(arrayBuffer);
+
+	let result = 0n;
+	let shift = 0n;
+	let index = 0;
+	try {
+		while (true) {
+			if (shift > 128n) {
+				return "";
+			}
+			const byte: bigint = BigInt(buf[index++]);
+			result |= (byte & 0x7Fn) << shift;
+			if ((0x80n & byte) === 0n) {
+				return result;
+			}
+			shift += 7n;
+		}
+	} catch {
```

Comment:
There shouldn't be any exceptions possible in this code, if we do the right bounds check. Can we remove the try/catch?

Reply:
Just in case the input buffer is too short and does not contain the expected final byte.

Reply:
Could we check for that case and handle it gracefully?